server/app/routes/resources/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+var router = require('express').Router(); // eslint-disable-line new-cap
+module.exports = router;
+
+const db = require('../../../db');
+const Resource = db.model('resource');
+const Tag = db.model('tag')
+
+
+//route would be /api/resources?type=article
+router.get('/', function(req, res, next) {
+    var reqType = req.query.type
+    if(reqType){
+       return Resource.findAll({
+        where:
+            {type: reqType},
+        include: [
+            {model: 'Tag'}
+        ]
+    })
+    .then(function(resources){
+        if (resources.length === 0){
+            res.status(404).send();
+        }
+        res.json(resources);
+    });
+    }
+    return Resource.findAll()
+    .then(function(resources) {
+        if (resources.length === 0){
+            res.status(404).send();
+        }
+        res.json(resources);
+    })
+    .catch(next);
+});
```

Comment:
This can be made simpler, with no if-else logic: just `findAll({ where: req.query })`. Works for finding everything, or finding by a query string.